doc/docs/master/concepts/pipeline-concepts/datum/index.md

```diff
@@ -5,63 +5,64 @@
     all data at once, each directory independently, individual
     files one by one, or combined data from multiple inputs together.
 
-A datum is the smallest indivisible unit of computation within a
-job.
+## Definition
+A datum is **the smallest indivisible unit of computation within a
+job**.
 A job can have one, many or no datums. Each datum is processed
-independently with a single execution of the user code and
-then the results of all the datums are merged together to
+independently with a single execution of the user code on one of
+the pipeline worker nodes.
+The results of all the datums are then combined together to
```

Comment:
`The results of all the datums` -> `The files output by all of the datums`. I'm being kind of nitpicky, but I think it is good to be very clear about it being the files output when processing the datum.